### PR TITLE
Feature additional storm urls

### PR DESCRIPTION
**Pull request by clindeman**

### CODE REVIEW

File: Python_env/hurrevac_storms.py [hunk 1]
```diff
@@ -95,12 +95,14 @@ def GetStormsJSON(self):
 ##        logging.debug("Running GetStormsJSON")
         manage = Manage()
         manage.handleProxy()
-        openUrl = requests.get(hurrevacSettings['HurrevacStormsURL'])
+        openUrl = requests.get(hurrevacSettings['HurrevacStormsURL'], timeout=10)
```
Comment (be-cunning) [inline Python_env/hurrevac_storms.py]: Why only 10 seconds? Is that enough time?  Also, should it re-try a few times (2 or 3 attempts) before officially going forward? We could add a while loop to try so.

File: Python_env/hurrevac_storms.py [hunk 2]
```diff
@@ -257,27 +259,49 @@ def GetStormJSON(self, StormId):
...
-                self.JSON = stormJSON
+                popupmsg("StormID not found.")
+                print(f"StormID data not found in {url_valid_list}")
+##              logging.warning("stormJSON length is 0, possible incorrect stormid")
         else:
             popupmsg("Error receiving data. Check settings.json url or site is down or changed.")
-            print("Error receiving data: %s" % openUrl.getcode())
-##            logging.error("Error receiving data: %s" % openUrl.getcode())
+            print("Error receiving data: %s" % openUrl.status_code)
+##          logging.error("Error receiving data: %s" % openUrl.status_code)
```
Comment (be-cunning) [inline Python_env/hurrevac_storms.py:304]: Can we remove this commented-out code?

File: Python_env/hurrevac_storms.py [hunk 3]
```diff
@@ -257,27 +259,49 @@ def GetStormJSON(self, StormId):
...
         Keyword Arguments:
            StormId :string -- Hurrevac stormid
+
+        Notes: 
+            https://hvx.hurrevac.com/hvx-api/v1/advisories/storm/{stormidhere has storms and
+            empty json for simulated storms.
+            https://hvx.hurrevac.com/hvx-api/v1/sim/advisories/storm/{stormidhere} has simulated storms and
+            empty json for non-simulated storms.
         """
 ##        logging.debug("Running GetStormJSON")
```
Comment (be-cunning) [inline Python_env/hurrevac_storms.py]: minor nit: why are the commented-out lines still here? That could cause some confusion down the road, plus, it makes the code less clean. Can we remove the commented-out lines?

File: Python_env/hurrevac_settings.json [hunk 4]
```diff
@@ -9,7 +9,10 @@
                     	"Exercise":0, 
                     	"Simulated":0},
 	"HurrevacStormsURL":"https://hvx.hurrevac.com/hvx-api/v1/advisories/storms", 
-	"HurrevacStormURL":"https://hvx.hurrevac.com/hvx-api/v1/advisories/storm",
```
Comment (be-cunning) [inline Python_env/hurrevac_settings.json:12]: could you just add the "/" at the end of each one of these? If so, then you wouldn't have to concatenate it later on.

**Review by be-cunning — DISMISSED** [review]

File: Python_env/hurrevac_settings.json (hunk 4 above)
Comment (clindeman) [reply Python_env/hurrevac_settings.json:12]: Yes, this will make the code cleaner.

File: Python_env/hurrevac_storms.py (hunk 2 above)
Comment (clindeman) [reply Python_env/hurrevac_storms.py:304]: We left the logging in as we were going to revisit, however we have yet to revisit after some time. There was an issue with the logging not working with the environment or the tool.

File: Python_env/hurrevac_storms.py (hunk 3 above)
Comment (clindeman) [reply Python_env/hurrevac_storms.py]: Yes, I've removed some commented-out lines.

File: Python_env/hurrevac_storms.py (hunk 1 above)
Comment (clindeman) [reply Python_env/hurrevac_storms.py]: Ten seconds should be enough time as the data is very small. Added code to try three times with two seconds in between each before raising an error.

**Review by be-cunning — APPROVED** [review]

**Pull request merged**